src/markdown.ts

```diff
@@ -1,8 +0,0 @@
-import remark from "remark";
-import html from "remark-html";
-import highlight from "remark-highlight.js";
```

Comment:
All these dependencies should removed from the package.json file. `remark-html` and `remark-highlight.js` have to be removed from the `src/@types` directory too